openapi.yaml

```diff
@@ -3497,8 +3500,739 @@ paths:
+              example:
+                products:
+                - type: Feature
+                  stac_version: 0.9.0
+                  id: c8a1f88d-89cf-4933-9118-45e9c1a5df20
+                  geometry: 
+                    type: Polygon
+                    coordinates: 
+                    - 
+                     - 
+                      - 12.36555287044679
+                      - 41.94403289260048
+                    -
+                      - 12.36571746774068
+                      - 41.86399361096971
+                    -
+                      - 12.60746759743069
+                      - 41.86372776276345
+                    -
+                      - 12.60758647471871
+                      - 41.94379931812686
+                    -
+                      - 12.36555287044679
+                      - 41.94403289260048
+                  properties:
+                    constellation: PHR
+                    acquisitionDate: "2022-03-21T10:11:15.055Z"
+                    azimuthAngle: 179.9852862071639
+                    cloudCover: 0
+                    geometryCentroid: 
+                      lat: 41.903935647240964
+                      lon: 12.486569672582828
+                    processingLevel: SENSOR
+                    sensorType: OPTICAL
+                    spectralRange: VISIBLE
+                  assets: {}
+                  links: []
+                links: []
+        4XX:
+          $ref: '#/components/responses/client_error_auth'
+        5XX:
+          $ref: '#/components/responses/server_error'
+      requestBody:
+        required: true
+        description: Data with desired filtering information to search for.
+        content:
+          application/json:
+            schema:
+              title: Search Commercial Products Request
+              type: object
+              required:
+                - collection_id
+                - parameters
+              properties:
+                collection_id:
+                  type: string
+                  description: Unique identifier of the collection.
+                parameters:
+                  type: object
+                  description: Key-value pairs of available `search_parameters` as listed at `GET /commercial_data/collections/{collection_id}` for filtering available products.
+                  additionalProperties:
+                    x-additionalPropertiesName: Search Parameter Name
+                    description: Value of the search parameter to be used in the request.
+              example:
+                collection_id: "PLEIADES"
+                parameters:
+                  bounds:
+                    - 3
+                    - 15
+                    - 4
+                    - 16
+                  time_range: 
+                    - "01-01-2022"
+                    - "01-02-2022"
+                  processing_level: SENSOR
+                  max_snow_coverage: 0
+  /commercial_data/orders:
```

Comment:
Any thoughts on aligning the general structure with https://github.com/stac-extensions/order ?

Reply:
It's ok, I used it but it's very limited in terms of information that it handles. Of course we could always extend it, but it's not quite clear yet what the best way of accessing commercial data would be, as mentioned in the description of this pull request (currently the solution works best for SH).

Reply:
I think we can loosely build on top of it.